android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerEvent.kt

```diff
@@ -51,5 +47,17 @@ class RNGestureHandlerEvent private constructor() : Event<RNGestureHandlerEvent>
       (EVENTS_POOL.acquire() ?: RNGestureHandlerEvent()).apply {
         init(handler, dataExtractor)
       }
+
+    @JvmStatic
+    fun <T: GestureHandler<T>> createEventData(
+      handler: T,
+      dataExtractor: RNGestureHandlerEventDataExtractor<T>?
+    ): WritableMap {
+      return Arguments.createMap().apply {
+        dataExtractor?.extractEventData(handler, this)
+        putInt("handlerTag", handler.tag)
+        putInt("state", handler.state)
+      }
+    }
```

Comment:
```suggestion
    ): WritableMap =
      Arguments.createMap().apply {
        dataExtractor?.extractEventData(handler, this)
        putInt("handlerTag", handler.tag)
        putInt("state", handler.state)
      }
    
```

Reply:
Fixed in 03580b23bbf7df785a6410dfa88443efca0e8b13